cdbot/cogs/fun.py

```diff
@@ -12,22 +12,23 @@
 import asyncpg
 from aiohttp import ClientSession
 from cdbot.constants import (
-    ADMIN_ROLES, EMOJI_LETTERS, FAKE_ROLE_ID, PostgreSQL, QUOTES_BOT_ID, QUOTES_CHANNEL_ID, STAFF_ROLE_ID,
-    WELCOME_BOT_ID
+    ADMIN_ROLES,
+    EMOJI_LETTERS,
+    FAKE_ROLE_ID,
+    PostgreSQL,
+    QUOTES_BOT_ID,
+    QUOTES_CHANNEL_ID,
+    STAFF_ROLE_ID,
+    WELCOME_BOT_ID,
 )
-from discord import Embed, File, Member, Message, NotFound
-from discord.ext.commands import Bot, Cog, Context, command, has_any_role
+from discord import Embed, File, Message, NotFound
+from discord.ext.commands import Bot, Cog, Context, UserConverter, command, has_any_role
 from discord.utils import find as discord_find
 from PIL import Image, ImageDraw, ImageFont
 
 ascii_lowercase += " !?$"
 
-REACT_TRIGGERS = {
```

Comment:
Why are we splitting some sections over multiple lines and not this one?

Reply:
It can be put in a single line without exceeding the 120 character line limit.

Reply:
For readability and easy expansion, this would be better split over multiple lines

Reply:
Will approve if changed.

Reply:
Done. The line limit should be made lower.